app/views/Dashboard/Overview/RegionalBreakdownCard/PieChartInfo/styles.css

```diff
@@ -0,0 +1,19 @@
+.pie-chart-wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 20%;
```

Comment:
why?

Reply:
because the re-charts container requires to be wrapped with a constraint